onnxruntime/test/optimizer/transpose_optimizer_test.cc

```diff
@@ -3711,7 +3711,10 @@ TEST(TransposeOptimizerTests, TestDequantizeLinearNoAxis) {
                     /*opset_version*/ 10);
 }
 
-TEST(TransposeOptimizerTests, TestDequantizeLinearTransposePropagation) {
+// TODO do we need this test and the associated transpose optimizer behavior of only swapping DQ/transpose when the
+// transpose is the only consumer now? see https://github.com/microsoft/onnxruntime/pull/12099.
+// the test doesn't work with the required EnsureUniqueDQForNodeUnit transformer.
+TEST(TransposeOptimizerTests, DISABLED_TestDequantizeLinearTransposePropagation) {
```

Comment:
not sure if we need this test. @yufenglee what do you think?

Reply:
Why this test doesn't work with the EnsureUniqueDQForNodeUnit  transformer?

Reply:
I made it run, but now it doesn't test what it was originally testing. this is because EnsureUniqueDQForNodeUnit will now duplicate the DQ for the second transpose.